docs/schema/snitch_cluster.schema.json

```diff
@@ -416,11 +416,11 @@
                                     "description": "Enable Smallfloat vector extension (SIMD).",
                                     "default": false
                                 },
-                                "snax": {
-                                    "type": "boolean",
-                                    "title": "SNAX Extension",
-                                    "description": "Enable Snitch Accelerator eXtension (SNAX)",
-                                    "default": false
+                                "snax_acc": {
+                                    "type": "number",
+                                    "title": "SNAX Accelerator Extension",
+                                    "description": "Enable specific Snitch Accelerator eXtension (SNAX). 1 for MAC engine and 2 for GEMM engine.",
+                                    "default": 0
```

Comment:
I don't like this schema; This way we cannot enable multiple accelerators being managed by one snax core. Do we want to rule this out right now already? Can we use a similar style of template to the hives? Where we add multiple templates for multiple accelerators? That makes more sense in my mind tbh:
e.g.
```hjson
snax_managed_accelerators{
      hwpe_accelerator_template,
      gemm_accelerator_template
}
```

Reply:
@JosseVanDelm Good question! I agree but no no no, we are not ruling it out

For now we need to set it to this 1 accelerator at a time first. I get your idea but it's not particularly easy to configure how align the TCDM ports with single SN core controlling multiple accelerators. I would like this to be in a separate PR because @xiaoling-yi, @guilherme-paim , and I need to study it carefully.  Basically easier said than done.

In this feature, I was actually thinking of making it more like a "solder" approach where there is more flexibility in creating a cluster shell. The current implementation is very "conditional" like "if (SNAX == 1) then generate MAC; if  (SNAX == 2) then generate GEMM" since we have less accelerators this should work for now. What I want is that we use the same "solder" mechanic where we have better connections to ANY accelerator. I don't also like the case where it's conditional. I prefer that our cluster shells are generated via a solder mechanic which we will learn next week :)

We were suppose to put it as an issue before you mentioned it haha. But yeah basically, we want the same feature but we need to think about the most elegant or configurable way to do this. This is just for now.